cmd/main.go

```diff
@@ -0,0 +1,102 @@
+// Copyright © 2024 Bank-Vaults Maintainers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
lets rename to `cmd` package to properly reflect the path
```suggestion
package cmd
```

Reply:
There would be no main package this way. :/

Reply:
i would keep the pattern, but move main yo cmd, and sync command to cmd/sync/sync.go

Reply:
I believe this is the current state.